culture-n-values.md

```diff
@@ -85,11 +85,11 @@ Alongside the books we read, we also devoted a lot of time to research other pla
 
 - [Google](https://rework.withgoogle.com/) - Thought it's organized more like a standalone site than properly a playbook, Google's Rework gives us quite a lot of insight in researches that they've done and how that affects their processes. This level of openness that shares even knowledge is one of the reasons Google is a reference in our field.
 
-- [Thoughtbot](https://thoughtbot.com/playbook) - Thoughtbot's playbook is structured to cover all the phases of their production process. Dividing it by phase makes it easier for people to go exactly where they want. Although focusing less on the internal operations of the company, their takes on sales, hiring, and apprenticeship are well worth the reading.
+- [Thoughtbot](https://thoughtbot.com/playbook) - Thoughtbot's playbook is structured to cover all the phases of their production process. Dividing it by phases makes it easier for people to go exactly where they want. Although focusing less on the internal operations of the company, their takes on sales, hiring, and apprenticeship are well worth the reading.
 
 - [Luminary Labs](http://www.slideshare.net/LuminaryLabs/the-human-company-playbook-version-10) - Before this playbook existed, Sara Holoubek did an excellent job advocating lengthy about how companies could go above the minimum line of benefits and start behaving more humanely. Recently they compiled not only their practices but also practices from multiple other companies. They are divided into sections that explain the importance of each benefit while listing many different ways that an interested company might implement them. It's a must read if you are interested in what benefits companies could offer.
 
-- [Facebook](http://v1.benbarry.com/project/facebooks-little-red-book) - Playbooks can also be a great tool for inspire people and align people with principles that you want for your company. Facebook's playbook does a very good job at that, focusing on the magnitude of tasks developed there and inspiring feelings of grandeur about changing the way people communicate. It doesn't talk much about internal practices, but it's a very good example of an inspirational playbook.
+- [Facebook](http://v1.benbarry.com/project/facebooks-little-red-book) - Playbooks can also be a great tool to inspire people and align them with principles that you want for your company. Facebook's playbook does a very good job on that, focusing on the magnitude of tasks developed there and inspiring feelings of grandeur about changing the way people communicate. It doesn't talk much about internal practices, but it's a very good example of an inspirational playbook.
```

Comment:
Any reason on the preference for ```on``` over ```at```? I think ```at``` fits better but both are fine. I would also add a ```by``` before the reasons why Facebook's playbook does a very good job at that. I would also remove the ```there```.

```suggestion
- [Facebook](http://v1.benbarry.com/project/facebooks-little-red-book) - Playbooks can also be a great tool to inspire people and align them with principles that you want for your company. Facebook's playbook does a very good job at that, by focusing on the magnitude of tasks developed and inspiring feelings of grandeur about changing the way people communicate. It doesn't talk much about internal practices, but it's a very good example of an inspirational playbook.
```